packages/brisa-tailwindcss/index.ts

```diff
@@ -49,7 +49,13 @@ export default function brisaTailwindcss({ embedded = true } = defaultConfig) {
           const to = path.join(destNodeModules, lib);
 
           if (await fs.exists(from)) {
-            return fs.cp(from, to, { recursive: true });
+            const toBinPath = path.join(to, 'node_modules/.bin');
```

Comment:
```suggestion
            const toBinPath = path.join(to, 'node_modules', '.bin');
```

This is to work on Windows that have a different separator than `/`.

Reply:
Uhh, windows. You are right!